cob_twist_controller/include/cob_twist_controller/constraints/constraint_ca_impl.h

```diff
@@ -287,7 +287,8 @@ void CollisionAvoidance<T_PARAMS, PRIO>::calcPartialValues()
         {
             if (params.frame_names.end() != str_it)
             {
-                Eigen::Vector3d collision_pnt_vector = it->nearest_point_frame_vector - it->frame_vector;
+                Eigen::Vector3d collision_pnt_vector = it->nearest_point_frame_vector;
```

Comment:
Well, I haven't seen the moveit-based distance computation fully finalized and pushed - including topic remaps and testing.
Has this been done? Or is it only locally working on your machine?